ecommerce/utils.py

```diff
@@ -55,3 +57,24 @@ def get_order_id_by_reference_number(*, reference_number, prefix):
         raise ParseException("Unable to parse order number")
 
     return order_id
+
+
+def send_support_email(subject, message):
+    """
+    Send an email to support.
+
+    Args:
+        subject (str): The email subject.
+        message (str): The email message.
+    """
+    try:
+        with mail.get_connection(settings.NOTIFICATION_EMAIL_BACKEND) as connection:
+            mail.send_mail(
+                subject,
+                message,
+                settings.MAILGUN_FROM_EMAIL,
+                [settings.EMAIL_SUPPORT],
+                connection=connection,
+            )
+    except:  # pylint: disable=bare-except
+        log.exception("Exception sending email to admins regarding enrollment failure")
```

Comment:
Should we remove the `enrollment failure` here since this function could be used in other circumstances?